### ISSUE TITLE
I cannot play some type of videos

### ISSUE THREAD

**Issue by rempas** [BrowserWorks/waterfox#2698]
### What happened?

After the last update, I'm not able to play some videos. Yeah, yeah, I know that there is an "info box" in the top of the window but there is no way to learn more and which codecs I need to install and where to find theme. This happened suddenly so I would expect some notes and info.

### Reproducible?

- [ ] I have tested this on Firefox.

### Version

G4

### What platform are you seeing the problem on?

Linux

**Comment by adamp01** [comment]
Hi - Is this version 4.1.1, 4.1.1.1, 4.1.2?

And can you give us some examples of videos you can no longer play?

**Comment by rempas** [comment]
The output of `waterfox --version` is:

`WaterfoxLimited Waterfox G4.1.2`

I'm using Arch Linux and I got Waterfox from AUR. Specifically, [this](https://aur.archlinux.org/packages/waterfox-g4-bin) package.

Some of the videos I cannot play (at least from the sites I tried until now) are YouTube liverstreams and any video from [gogoanime](https://gogoanime.run).

**Comment by adamp01** [comment]
OK thanks, let me see if I can reproduce.

**Comment by adamp01** [comment]
I can't reproduce this locally. @hawkeye116477 - have you noticed anything like the above with the Arch package?

**Comment by hawkeye116477** [comment]
That mentioned package is basically same as official (tarball+few tweaks). Anyway, everything is fine on my side.

**Comment by rempas** [comment]
@adamp01 @hawkeye116477 Hm, weird. I thought, you explicitly changed the functionality and now extra codecs are needed.

Specifically, when I go on some sites, I get a box saying: `To play video, you may need to install the required video codecs.`

This is part of the Waterfox's UI and it is in the bottom after the url box and above the start of the webpage. It is weird because this showed up suddenly after an update on Waterfox and I didn't removed anything in my system so I don't really know how to help here.

Is there a way to learn which codes are required for playing videos on Waterfox in general so I can learn and find which one's I have?

**Comment by ladytintagel** [comment]
I am not sure if there has been any movement here, but I am having the exact same issue.  I get the `To play video, you may need to install the required video codecs.` message on random sites.  Videos in many places work, such as youtube, but videos on Facebook, for instance, do not.  

I also installed waterfox in Arch from the AUR. 

Output of waterfox-g4 --version:
`WaterfoxLimited Waterfox G4.1.2`

**Comment by ghost** [comment]
Same issue here.
I'm using the package "waterfox-g4-kpe" in Fedora 36.

My --version output:
`WaterfoxLimited Waterfox G4.0.7`

**Comment by ghost** [comment]
I was able fix the issue by installing "compat-ffmpeg28" on Fedora.
Apparently, you can fix on Arch as well by installing "ffmpeg-compat-58".

**Comment by rempas** [comment]
@MatNascimento Thank you!!!

@adamp01 I think that there should be a message that the "ffmpeg-compat" is needed.

So the message: `To play video, you may need to install the required video codecs.`
can become: `To play video, you may need to install the "ffmpeg-compat" package.`

This way, users will know.

**Comment by Audition-CSBlock** [comment]
> @adamp01 I think that there should be a message that the "ffmpeg-compat" is needed.
> 
> So the message: `To play video, you may need to install the required video codecs.` can become: `To play video, you may need to install the "ffmpeg-compat" package.`
> 
> This way, users will know.

This would only be valuable to certain distro's or os's, and ffmpeg-compat is named differently in fedora as @MatNascimento mentioned.

**Comment by rempas** [comment]
> This would only be valuable to certain distro's or os's, and ffmpeg-compat is named differently in fedora as @MatNascimento mentioned.

Oh, I see!

**Comment by hawkeye116477** [comment]
It's probably this problem => https://bugzilla.mozilla.org/show_bug.cgi?id=1750760, means that we need support for ffmpeg 5.

**Comment by rempas** [comment]
@hawkeye116477 When you say "we", who exactly do you refer too? Waterfox or Firefox's core?

**Comment by hawkeye116477** [comment]
It's already fixed on FF, but 99, so maybe eventually Waterfox team could backport that.

**Comment by rempas** [comment]
Nice! In that case, I'll let the Waterfox team close this when they can confirm that it's fixed for everyone!

**Comment by Audition-CSBlock** [comment]
@rempas I used the same package from the aur as you did and it still doesn't work for me. I tried with clean profiles, and downgrading ffmpeg also didn't seem to fix the problem. Now i'm using librewolf which works fine for me so far, good luck with the issue :sweat_smile:

**Comment by rempas** [comment]
@Audition-CSBlock Yeah, I moved to librewolf as well. I hope the team can fix the issue in the future!

**Comment by allanfherrera** [comment]
Try pasting the Link of the video into VLC Media Player and it will tell you what codecs you need in the Tools> Codec Information section.
